CRM/Extension/System.php

```diff
@@ -138,6 +138,14 @@ public function getFullContainer() {
           );
         }
       }
+      // I've pointed this at the Financial directory for now as I think we probably want to group them
+      // but don't want to go too far until agreed.
+      $containers['coreextensions'] = new CRM_Extension_Container_Basic(
+        CRM_Utils_File::addTrailingSlash($this->parameters['civicrm_root'], DIRECTORY_SEPARATOR) . 'CRM/CoreExtensions/Financial',
+        CRM_Utils_File::addTrailingSlash($this->parameters['resourceBase'], '/') . 'CoreExtensions/Financial',
+        $this->getCache(),
+        'coreextension'
+      );
```

Comment:
Aside: Was this added because of an empirical/observed issue or an anticipated issue?

The reason I ask... the `$civicrm_root` is already a basic container (lines 117-122), and basic containers are searched recursively, so anything under `$civicrm_root` should already be searched. This is how `civicrm/ext/` and `civicrm/tools/extensions/` are loaded. My expectation is that this snippet shouldn't be needed.

Reply:
Oh - right - yeah - I should try removing it again then!

Reply:
So in that case the only difference between ext & this new location is the new location is unequivocally part of core & in the core repo - whereas ext is maintained by repo maintainers (currently only one )

Reply:
@totten you are correct - I've removed this change